CHANGELOG.md

```diff
@@ -2,6 +2,21 @@
 
 All notable changes to the "Azure Api Management VS Code" extension will be documented in this file.
 
+## 1.1.0
+
```

Comment:
Should we add date under the release version?
I saw this other release notes doing as well. but up to you

Reply:
The challenge here is we do not know when the extension is published at the time being. Since VS Code marketplace provides the accurate release time, I didn't include a date anymore. The APIC extension also did not include a date.
Feel free to ping me to discuss more. I'm going to merge it first to kick off the test.

Reply:
ok, that makes sense - works for me then :)